### PR TITLE
Link fixes

### PR DESCRIPTION
**Pull request by jomey**
Point some outdated links from the template to either the updated template URL or the official eScience site.

### CODE REVIEW

**Comment by aaarendt** [comment]
Not sure if I'm missing something, but the preview version of the Jupyter Book no longer has sidebar links to schedule and team. Wasn't sure if that was intentional as the code seems to indicate otherwise?

**Comment by jomey** [comment]
> Not sure if I'm missing something, but the preview version of the Jupyter Book no longer has sidebar links to schedule and team. Wasn't sure if that was intentional as the code seems to indicate otherwise?

That is definitely a bug! Let me look into it

**Comment by jomey** [comment]
>
> That is definitely a bug! Let me look into it

@aaarendt - Great catch and now the links are properly fixed :smiley:

File: book/projects/index.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+```{image} ../img/project-work.png
+:alt: picture of people interacting during hackweek project work
+:class: bg-primary mb-1
+:width: 500px
+:align: center
+```
+
+## Projects from Past Events
+
+Visit our [gallery of past hackweeks](https://guidebook.hackweek.io/hackweeks-as-a-service/gallery/index.html) to browse our catalog of amazing projects from previous events!
```
Comment (JessicaS11) [inline book/projects/index.md]: This link is a 404. the problem with the [escience version](https://escience.washington.edu/using-data-science/hackweeks/archive/) is it doesn't yet include the ICESat-2 Hackweeks...

File: book/team.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ description:
 
 # INSTRUCTIONS:
 # - One entry per key must be included per person (title, avatar, role, organizations, bio, expertise, social, user_groups)
-# - user_groups correspond to hackweek roles defined here: https://uwhackweek.github.io/hackweeks-as-a-service/services/index.html
+# - user_groups correspond to hackweek roles defined here: https://guidebook.hackweek.io/hackweeks-as-a-service/services/index.html
```
Comment (JessicaS11) [inline book/team.yaml]: also a 404. Seems to be the guidebook.hackweek.io links.

File: book/projects/index.md (hunk 1 above)
Comment (jomey) [reply book/projects/index.md]: Another great find and thorough check of the links :+1: ... Fixed

**Comment by jomey** [comment]
Should we add the link to the CryoHub on the [Jupyter Book index page](https://deploy-preview-14--icesat-2-2023.netlify.app/intro.html) with this PR?

**Review by aaarendt — APPROVED** [review]

**Pull request merged**